<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmList</class>
 <widget class="QDialog" name="AlarmList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm List</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>641</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
